src/components/pages/Mediation/Mediation.jsx

```diff
@@ -400,7 +401,7 @@ class Mediation extends PureComponent {
         </p>
         <p>
           {'Utilisateurs avancés : vous pouvez '}
-          <a href="https://pass.culture.fr/assets/docs/PassCulture-accroche-template-20181114.zip">
+          <a href={ASSETS_URL + '/PassCulture-accroche-template-20181114.zip'}>
```

Comment:
```suggestion
          <a href={`${ASSETS_URL}/PassCulture-accroche-template-20181114.zip`}>
```
Ecriture plus moderne.